client/settings/views/wcc-settings-form/settings-item.js

```diff
@@ -102,20 +100,15 @@ const SettingsItem = ( {
 			);
 
 		case 'packages':
-			const packagesState = form.packages;
+			const packagesMsg = sprintf(
+				__( 'Manage your packages on <a href="%(url)s">the packages settings page</a>' ),
+				{
+					url: '/wp-admin/admin.php?page=wc-settings&tab=connect&section=packages',
+				}
+			);
+
 			return (
-				<Packages
-					{ ...packagesState }
-					{ ...packagesActions }
-					packages={ fieldValue }
-					presets={ schema.definitions.preset_boxes }
-					dimensionUnit={ storeOptions.dimension_unit }
-					removePackage={ removeArrayItem }
-					savePackage={ savePackage }
-					weightUnit={ storeOptions.weight_unit }
-					errors={ errors }
-					packageSchema={ fieldSchema.items }
-				/>
+				<div dangerouslySetInnerHTML={ { __html: packagesMsg } } />
```

Comment:
This link gives a `404` in my machine, because I have WordPress installed in a sub-folder (`localhost/wordpress/wp-admin...`). The link should be just `admin.php?page=wc-settings&tab=connect&section=packages`